src/index.tsx

```diff
@@ -36,7 +36,7 @@ export class ApptentiveConfiguration {
     this.apptentiveSignature = signature;
     this.logLevel = "info";
     this.distributionName = "React Native";
-    this.distributionVersion = require("../package").version;
+    this.distributionVersion = "6.7.0";
```

Comment:
I will remove this as well. I was going through an issue Rattlesnake not able to find package.version though it was just there.